lxc/storage.go

```diff
@@ -124,13 +144,21 @@ func (c *cmdStorageCreate) run(cmd *cobra.Command, args []string) error {
 	pool.Config = map[string]string{}
 	pool.Driver = args[1]
 
-	for i := 2; i < len(args); i++ {
-		entry := strings.SplitN(args[i], "=", 2)
-		if len(entry) < 2 {
-			return fmt.Errorf(i18n.G("Bad key=value pair: %s"), entry)
+	if stdinData.Config == nil {
+		for i := 2; i < len(args); i++ {
+			entry := strings.SplitN(args[i], "=", 2)
+			if len(entry) < 2 {
+				return fmt.Errorf(i18n.G("Bad key=value pair: %s"), entry)
+			}
+
+			pool.Config[entry[0]] = entry[1]
 		}
+	} else {
+		pool.Config = stdinData.Config
+	}
 
-		pool.Config[entry[0]] = entry[1]
+	if stdinData.Description != "" {
```

Comment:
This is unnecessary, please follow the style for `cmdNetworkForwardCreate` (as an example).

The default for `Description` is empty, so we can just use it directly from stdinData.